web_widget_table/readme/USAGE.rst

```diff
@@ -0,0 +1,37 @@
+Use this widget by saying:
+
+.. code-block:: xml
+
+    <field name="my_table" widget="widget_table" options="{
+                                'field_name': 'child_ids',
+                                'datatable_params':{'searching': False,
+                                'paging': False,
+                                'info': False},
+                                'fields': ['name','date_start', 'date_end', 'real_date_start', 'real_date_end', 'total_time'],
+                                'headers': ['Name','Date Start', 'Date End', 'Real Date Start', 'Real Date End', 'Total Time']}"/>
```

Comment:
headers should be optional and come by default from field names, especially because here you cannot handle translations.